packages/integrations/cloudflare/src/index.ts

```diff
@@ -203,7 +203,7 @@ export default function createIntegration(args?: Options): AstroIntegration {
 				}
 
 				if (isModeDirectory) {
-					const functionsUrl = new URL(`file://${process.cwd()}/functions/`);
+					const functionsUrl = new URL(`file://${_config.root}/functions/`);
```

Comment:
This is not working for me. `_config.root` is a URL itself.
Changing this line to
`const functionsUrl = new URL("functions", _config.root);`
works for me, also the test is running fine than.

Reply:
Awesome, that helps! Thanks for the feedback! Making the changes now.

Reply:
I made this change, but tests keep failing locally for me. I will push the update so we can have the same reference.

Reply:
It does seem to work, interesting. 

Thank you :)